### PR TITLE
Feature #16 : Setup autoUpdater

### PR DESCRIPTION
**Pull request by leohubert**

### CODE REVIEW

**Review by iWirk — APPROVED** [review]
Great job!

File: src/renderer/pages/servers/_id.vue [hunk 1]
```diff
@@ -293,13 +293,13 @@ export default class Server extends Vue {
...
-        alert('Empty modpack, please contact an administrator...')
+        alert('Empty ModPack, please contact an administrator...')
         return
       }
 
-      const downloader = await this.module.createDownloader(this.server.id, modPacks) as Downloader
+      await this.module.createDownloader(this.server.id, modPacks)
 
-      await downloader.start(this.forceUpdate)
+      await this.downloader.downloader.start(this.forceUpdate)
```
Comment (leohubert) [inline src/renderer/pages/servers/_id.vue:302]: this.downloader.downloader ? `not beautiful`

File: src/shared/communicator/main/MainCommunicator.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    })
+  })
+
+  ipcMain.on(channel, (event, prop: string) => {
+    console.log(channel, prop)
+    event.returnValue = currentInstance[prop]
+  })
+
+  ipcMain.handle(`${channel}-call`, async (_, prop: string, ...args: any[]) => {
+    console.log(`${channel}-call`, prop, args)
```
Comment (leohubert) [inline src/shared/communicator/main/MainCommunicator.ts:71]: cleanup console.log

File: src/shared/communicator/renderer/RendererCommunicator.ts [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+          }
+        })
+      }
+
+      call (func: any, ...args): any {
+        return ipcRenderer.invoke(`${this.channel}-call`, func, ...args)
+      }
+
+      destroy () {
+        console.log('destroy')
```
Comment (leohubert) [inline src/shared/communicator/renderer/RendererCommunicator.ts:90]: console.log

**Pull request merged**